modules/rosa-rn-security.adoc

```diff
@@ -0,0 +1,55 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-security_{context}"]
+= Security
+
+[id="rosa-rn-auth-provider_{context}"]
+== Authentication provider
+
+Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] cluster creation process or using the `rosa` CLI.
+
+OpenShift Container Platform is not an identity provider, and all access to the cluster must be managed by customers as part of their integrated solution. {product-title} (ROSA) supports the use of multiple identity providers that are simultaneously provisioned.
+
+The following identity providers are supported:
+
+* GitHub or GitHub Enterprise
+* GitLab
```

Comment:
@wgordon17 - Is GitLab supported on ROSA?  I don't think it is on OSD...

Reply:
Yup, I see it in the ROSA CLI

```
Type of identity provider. Options are [github gitlab google ldap openid]
```

Reply:
Thanks for checking. I'll leave it as is then.